src/screens/codeVerification/hooks/useCodeVerification.ts

```diff
@@ -70,13 +70,9 @@ const useCodeVerification = (): TCodeVerificationHook => {
 
   useEffect(() => {
     if (generateCodeError)
-      if (
-        generateCodeError !==
-        'Você possui um código ativo,so pode ser gerado outro após 1 minuto.'
-      )
-        setErrorMessage(
-          'Falha ao enviar o código. Volte ao login e tente novamente.'
-        );
+      setErrorMessage(
+        'Falha ao enviar o código. Volte ao login e tente novamente.'
+      );
```

Comment:
Funcionou mas quando entro na tela de verificação já abre com um erro indicando sair da tela. O ideal seria não mostrar esse erro quando o problema for ja ter codigo ativo msm

<img width="519" alt="Captura de Tela 2023-08-24 às 11 14 08" src="https://github.com/ajuda-ai-ufam/frontend/assets/26875510/0064536c-d9bd-468c-b67d-d6f81ff8ddba">